jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/exporter/EntityExporter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.gridexportflowui.exporter;
+
+import io.jmix.gridexportflowui.exporter.entitiesloader.AllEntitiesLoader;
+
+/**
+ * Visitor is passed to {@link AllEntitiesLoader} to export loaded entity
+ */
+public interface EntityExporter {
```

Comment:
1. In the javadoc you explicitly says that this a **visitor**. So why not to name the class **visitor**? It doesn't actually do the export. Something like `ExportedEntityVisitor` with the `visitEntity()` method? Another argument for renaming this class is that it kind of conflicts with `JsonExporter` and `ExcelExporter`, although it has nothing to do with them.
2. Why this class in the top-level package `io.jmix.gridexportflowui.exporter` although it is used only by classes from the `io.jmix.gridexportflowui.exporter.entitiesloader` package. This class may be moved to the nested package.
3. If we go on to develop the idea of moving this interface, then what about making it the nested interface of `AllEntitiesLoader`?